scripts/gen_vimdoc.py

```diff
@@ -54,6 +59,18 @@
     print("requires Python 3.5+")
     sys.exit(1)
 
+if not shutil.which('doxygen'):
+    print("Missing Requirement: doxygen")
```

Comment:
I realize this is following the existing pattern, but error messages should really be going to stderr.

```suggestion
    print("Missing Requirement: doxygen", file=sys.stderr)
```

Reply:
Cool, I will change to do that in the other places as well.